ledfx/api/websocket.py

```diff
@@ -297,6 +300,36 @@ def audio_stream_data_handler(self, message):
             message.get("data").values(), dtype=np.float32
         )
 
+    @websocket_handler("audio_stream_data_v2")
+    def audio_stream_data_base64_handler(self, message):
+        # Max value for signed 16-bit values.
+        MAX_VAL = 32767
```

Comment:
Can you please make this a global constant in this file to get it out of the function - unlikely to be causing any performance issues but if its not changing it belongs as a const.